README.md

```diff
@@ -7,3 +7,19 @@ This app will replace the weekly 'events briefs' that are prepared by instructor
 ## Tech Stack
 
 The site is built in Sinatra and stores data in a CSV file.
+
+## Stories
+### Story 1 -- Find an event and RSVP
+
+Susie should be able to go to the homepage and the events for the current week will be displayed. The current week will be the title and the events will be organized by weekday and then time, begins on Monday. Each event will link to an event page. Susie can click on the link of the event that interests her. This takes her to a new view that displays the event information: event name, group name, date of event, time of event, event location, link to original/meetup page. Additionally, there is a list of all attendees, which are all the other people that have already rsvped. Lastly there is a form in which Susie can add her name to RSVP. Susie will type her name into the text input box hit submit and her name will be added to the attendee list.
```

Comment:
Try to define more granular stories. Each story should be kind of like a Toy Problem: Specific input/action and specific output/consequence.

In this first story, I see a few stories:

1. Susie should be able to go to the homepage and the events for the current week will be displayed. The current week will be the title and the events will be organized by weekday and then time, begins on Monday.
2. Each event will link to an event page. Susie can click on the link of the event that interests her. This takes her to a new view that displays the event information: event name, group name, date of event, time of event, event location, link to original/meetup page.
3. Additionally, there is a list of all attendees, which are all the other people that have already rsvped.
4. Lastly there is a form in which Susie can add her name to RSVP. Susie will type her name into the text input box hit submit and her name will be added to the attendee list.